new-site/src/components/seo.js

```diff
@@ -1,51 +1,33 @@
 /**
- * SEO component that queries for data with
- *  Gatsby's useStaticQuery React hook
+ * SEO component
  *
  * See: https://www.gatsbyjs.com/docs/use-static-query/
  */
 
-import * as React from 'react';
-import PropTypes from 'prop-types';
 import { Helmet } from 'react-helmet';
-import { useStaticQuery, graphql } from 'gatsby';
-
-function Seo({ description, lang, meta, title }) {
-  const { site } = useStaticQuery(
-    graphql`
-      query {
-        site {
-          siteMetadata {
-            title
-            description
-          }
-        }
-      }
-    `,
-  );
-
-  const metaDescription = description || site.siteMetadata.description;
-  const defaultTitle = site.siteMetadata?.title;
+import PropTypes from 'prop-types';
+import * as React from 'react';
 
+function Seo({ lang, meta, title, description, pageTitle }) {
   return (
     <Helmet
       htmlAttributes={{
         lang,
       }}
       title={title}
-      titleTemplate={defaultTitle ? `%s | ${defaultTitle}` : null}
+      titleTemplate={title ? `${title} | ${pageTitle}  ` : null}
```

Comment:
Do we need the titleTemplate? I tried to study what it means, and it appears to be used in a case where react-helmet's `Helmet` component is used twice; the first usage has the titleTemplate defined, and then the second usage provides only title, which is then appended to the titleTemplate. Like in this example: https://dev.to/guimg/dynamic-document-head-with-react-helmet-28o6

I could not find the second usage of the `Helmet` component in this repository. Am I missing something, or have I understood the usage wrong?

Reply:
- [ ] simplify title fields